functions/tab.fish

```diff
@@ -100,8 +100,12 @@ Arguments:
       tab.gnome_terminal "$cdto" "$cmd"
 
     case "*"
-      echo "Unknown terminal: $term_program" >&2
-      return 1
+      if test -n "$GUAKE_TAB_UUID"
```

Comment:
rather than adding special handling in the `*` case, update `__tab.term_program` to detect `$GUAKE_TAB_UUID` and return `guake` :)